docs/conf.py

```diff
@@ -73,28 +73,30 @@ def _compute_navigation_tree(context):
 # domain name if present. Example entries would be ('py:func', 'int') or
 # ('envvar', 'LD_LIBRARY_PATH').
 nitpick_ignore = [
-    ('py:class', 'ops.model._AddressDict'),
+    # Please keep this list sorted alphabetically.
```

Comment:
Note that the only thing I've added here is `_NoticeDict`, but I sorted the list to make the it easier to scan.